whitesource/run_whitesource.sh

```diff
@@ -1,31 +1,39 @@
 #!/usr/bin/env bash
 #
 # Run whitesource for components which need versioning
-
-set -e
-set -o pipefail
+# Catch error and exit only if not -2 or 0
+#
+# If an error occurs, return an error code instead of exit -1
+set +e
+# Fail if any command in pipe fails.
+set -o pipefail 
 
 cd ${WORKSPACE}
 
 # SCAN_DIRECTORIES is a comma-separated list (as a string) of file paths which contain all source code and build artifacts for this project
 SCAN_DIRECTORIES=$PWD
 
+# If your PROD_BRANCH is not master, you can define it here based on the need
+PROD_BRANCH="master"
+
 # PRODUCT_NAME is your team's name or overarching project name
 PRODUCT_NAME="spark-snowflake"
 
```

Comment:
I suspect GIT_BRANCH is not set. Could you please set it and echo it? 
export GIT_BRANCH=`git branch -l | grep “\*” | awk ‘NF>1{print $NF}’ | cut -d”)” -f1`
echo $GIT_BRANCH